src/script/components/MessagesList/Message/ContentMessage/asset/LinkPreviewAssetComponent.tsx

```diff
@@ -116,13 +116,14 @@ const LinkPreviewAsset: React.FC<LinkPreviewAssetProps> = ({header = false, mess
                 <p>{t('conversationTweetAuthor')}</p>
               </div>
             ) : (
-              <p
+              <a
                 className="link-preview-info-link text-foreground ellipsis"
+                href={preview.url}
```

Comment:
clicking the link preview (or that particular part that you modified) should open a new tab (not navigate out of Wire to that link).

```suggestion
                href={preview.url}
                target="_blank"  
                rel="noopener noreferrer"
```